.github/workflows/lint.yml

```diff
@@ -46,17 +23,10 @@ jobs:
         env:
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
 
-      - run: npm run lint --if-present
+      - run: make lint
+      - run: make format
 
-  # This job just check code style for in-template contributions.
-  code-style:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
-      - uses: actions/setup-node@v4
+      - name: commit
```

Comment:
I wouldn't run the `prettier/eslint --write` in the pipeline and commit it without a proper review. 

What do you think about implementing something like https://github.com/lint-staged/lint-staged + https://typicode.github.io/husky/ in a future PR?